recipes/quickstart/inference/local_inference/README.md

```diff
@@ -7,6 +7,32 @@ The way to run this would be
 ```
 python multi_modal_infer.py --image_path "./resources/image.jpg" --prompt_text "Describe this image" --temperature 0.5 --top_p 0.8 --model_name "meta-llama/Llama-3.2-11B-Vision-Instruct"
 ```
+---
+## Multi-modal Inferencing Using Gradio UI for inferencing
+For multi-modal inferencing using Gradio UI we have added [multi_modal_infer_Gradio_UI.py](multi_modal_infer_Gradio_UI.py) which used Gradio and transformers library.
+
+### Steps to Run
+
+1. **Set Up Hugging Face Token**:
+   - Export your Hugging Face token with proper access to the Llama 3.2 vision models:
+     ```
+     export hf_token=<your-hf-token>
```

Comment:
To make it consistent with the inference recipe-can you please pass this is an arg?

Reply:
Adding it in a while

Reply:
done with adding passing hugigng-face token from arguments